src/PercentDiscountPurchase.java

```diff
@@ -0,0 +1,27 @@
+import java.util.Scanner;
+
+public class PercentDiscountPurchase extends Purchase {
+    public static final int NUMBER_PURCHASE_FORE_DISCOUNT = 20;
+    public static final double DISCOUNT = 25;
```

Comment:
not a constanse according to the guide (but I choose just the same way for my first variant)